.github/ISSUE_TEMPLATE/decommissioning_v_1_0.md

```diff
@@ -0,0 +1,58 @@
+
+>**Servername :** <!-- Servername to decommission -->  
+>**FQDN:** <!-- FQDN -->  
+>**IP:** <!-- IP -->  
+>**Contact person:** <!-- Contact person -->  
+>**Projectname:** <!-- Name of the project -->  
+  
+- [ ] System in PRTG stop with a comment (ID-Service Now)
+- [ ] Revoke SSL certificate. Assignment group "ID - PKI" ID <!-- put the ID Number here -->
+      <p>ID-Service Now: <input type="text" size="20" maxlength="40" name="ID-Number" value="ID-Number" /></p>  
+- [ ] Delete DNS Entries (if possible), otherwise create a Ticket in Service Now to the assignment group "ID - Hostmaster" (Technikcal Service: DDI (DNS/DHCP/IP)) with the request to delete the Host entry:
+         <p>ID-Service Now: <input type="text" size="20" maxlength="40" name="ID-Number" value="ID-Number" /></p>  
+    **DNS list**
+  > - [ ] <!-- <FQDN> and <IP> -->
+  
+  > - [ ] <!-- <FQDN> and <IP> -->  
+
+  > - [ ] <!-- <FQDN> and <IP> -->
+
+- [ ] Delete AD entry if server im AD (sssd auth)
+- [ ] Delete PRTG entries
+- [ ] Remove host from configuration management (Ansible) 
+- [ ] Remove Database from configuration management (Ansible)
+- [ ] Delete any email accounts
+- [ ] Cancel Backup (Tape). Email to Büren, Peter (ID) <peter.vonbueren@unibe.ch> 
+- [ ] In the Secret Server (if possible) delete (AKA deactivate) passwords and information that are no longer used. (https://idsecret.unibe.ch/SecretServer)
+- [ ] Firewall-Rules: create a Ticket in Service Now to assignment group "ID - Security" (Technical Service: Firewalldienste) with Hostname/IP with the request to delete all firewall rules 
+       <p>ID-Service Now: <input type="text" size="20" maxlength="40" name="ID-Number" value="ID-Number" /></p>   <!-- put the ID Number here -->
+- [ ] Elasticsearch: create a Ticket in Service Now to assignment "ID - Security" (Technical Service: Elasticsearch) with the request to delete the host from elasticsearch. 
+        <p>ID-Service Now: <input type="text" size="20" maxlength="40" name="ID-Number" value="ID-Number" /></p> <!-- put the ID Number here -->
+- [ ] Check documentation to see if there are any references to other dependencies that need to be deleted
+- [ ] Delete any documentation for the System
+- [ ] Inform those responsible for the system and ask them to delete relevant documentation in their space in confluence.
+- [ ] Database in PostgreSQL will also be deleted if it exists. 
+- [ ] NAS configs will be deleted. Send a Email with the Servername and IP to Roland, Trummer <roland.trummer@unibe.ch> and wait the confirmation
+- [ ]  Shut des Systems:
+> - [ ] - a.) If ESXI VM cannot be deleted → create a Ticket to ESXI Team with the request to delete the server
+> - [ ] - b.) If hardware system → remove the disks, wipe them and pass them on correctly place (ESXI Büro in die Blaue Box), Check the inventory number and check the next steps with Lead SYS (disposal / Storage)
+- [ ] Delete Server
```

Comment:
```suggestion
- [ ] Delete the server:
  - [ ] a.) delete VM in vCenter, if VM cannot be deleted → create a Ticket to ESXI Team with the request to delete the server
  - [ ] b.) If hardware system → remove the disks, wipe them and pass them on correctly place (ESXI office into the blue box), Check the inventory number and check the next steps with Lead SYS (disposal / Storage)

```

Not sure if the blue box is still correct. Could also be elsewhere nowadays (office WPS, cellar, ..?)

Reply:
I asked Roland Trummer. There is no longer a blue box. If the disk has been wiped, dispose of it in the container in the basement, where the electronic waste is.